compare/README.md

```diff
@@ -5,11 +5,12 @@ To build a test utility configure libass with the `--enable-compare` flag.
 The utility works with `png` image files so there is external dependency of libpng.
 
 Test program command line:  
-`compare ([-i] <input-dir>)+ [-o <output-dir>] [-s <scale:1-8>]`
+`compare ([-i] <input-dir>)+ [-o <output-dir>] [-s <scale:0-8>]`
```

Comment:
imho strict/detailed mode should be a separate flag.

Reply:
I've created pass-level flag, which encompasses an old strict mode.